lib/dish.rb

```diff
@@ -0,0 +1,28 @@
+class Dish
+  attr_reader :dishes
+
+  def initialize
+    @dishes = []
+
+    preload_dishes
+  end
+
+  def print
+    return 'Puts no students at this time' if @dishes.empty?
+    i = 1
+    @dishes.each do |dish|
+      "#{i}. #{dish[:name].capitalize.ljust(24, '.')} #{dish[:price]}"
```

Comment:
What is this #each block supposed to do? Remember that #each returns the array it is called on.